cirq/google/devices/serializable_device.py

```diff
@@ -28,19 +28,43 @@
 class _GateDefinition:
     """Class for keeping track of gate definitions within SerializableDevice"""
 
-    def __init__(self, duration: 'cirq.DURATION_LIKE',
-                 target_set: Set[Tuple['cirq.Qid', ...]], number_of_qubits: int,
-                 is_permutation: bool):
+    def __init__(
+            self,
+            duration: 'cirq.DURATION_LIKE',
+            target_set: Set[Tuple['cirq.Qid', ...]],
+            number_of_qubits: int,
+            is_permutation: bool,
+            can_serialize_function: Callable[['cirq.Gate'], bool] = lambda x:
```

Comment:
On serializers this is called `can_serialize_predicate`. Might be good to be consistent with that.